app/assets/stylesheets/overlays/_paper_new_overlay.scss

```diff
@@ -1,11 +1,10 @@
-.paper-new-overlay .chosen-style-override .chosen-container {
-  width: 44% !important;
-}
-
-.paper-new-overlay .overlay-container {
-  width: 790px;
-}
+.paper-new-overlay {
+  .chosen-style-override .chosen-container {
+    width: 44% !important;
```

Comment:
!important should not be used<br>Properties should be ordered margin-bottom, width